docs/introduction/faq-and-common-issues.md

```diff
@@ -205,8 +205,8 @@ To resolve this issue, you can modify your Git configuration to allow long paths
 
 Order total price needs to be configured manually in template content in administration `Settings -> Communication with customer -> E-mail templates`.
 Configuration for transport, payment and product prices can be easily changed using `shopsys.mail_template.display_price` parameter in `parameters_common.yaml`.
-Supported values are `input_price` and `both` when both stands for both price with and without VAT.
-The default is `input_price` which displays the price using setting of the input price type in the administration.
+Supported values are `selling_price` and `both` when both stands for both price with and without VAT.
```

Comment:
wow, I did not even know we have such setting :smile:

Reply:
we should mention the change in the upgrade notes, right? Because the following setting won't work anymore, right?
```
shopsys.mail_template.display_price: 'input_price'
```

It might be covered by see-project-base-diff but I still think it deserves the mention  :slightly_smiling_face: